src/framework/vst/internal/vstplugin.cpp

```diff
@@ -172,7 +172,7 @@ void VstPlugin::stateBufferFromString(VstMemoryStream& buffer, char* strData, co
 
     static Steinberg::int32 numBytesRead = 0;
 
-    buffer.write(strData, strSize, &numBytesRead);
+    buffer.write(strData, static_cast<size_t>(strSize), &numBytesRead);

```

Comment:
No you need to cast (_from_ `size_t`) _to_ `Steinberg::int32`

Reply:
Oh boy... senior moment I guess